frontend/src/app/core/helpers/api.ts

```diff
@@ -1,8 +1,8 @@
 export const api = <T>(url: string, init?: RequestInit): Promise<T> => {
     return fetch(url, init)
-        .then(response => {
+        .then(async response => {
             if (!response.ok) {
-                throw new Error(response.statusText)
```

Comment:
[`statusText`](https://developer.mozilla.org/en-US/docs/Web/API/Response/statusText) is just a description of the status code like "OK", "ERROR".  This PR uses the response body instead.